<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cam_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>cam_img</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>480</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
     </font>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <double>0.050000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="text_delay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_time_per_step">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>time per step</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_per_step">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{margin-top:-15px; border:none;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QRadioButton" name="right_hand">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>D2Coding</family>
      </font>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="left_hand">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>D2Coding</family>
      </font>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="text_hand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>hand</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_ctr_area">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>control area</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ctr_area_ratio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <double>0.050000000000000</double>
    </property>
    <property name="maximum">
     <double>0.800000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.800000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="help_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar{border :2px solid rgb(186, 189, 182)}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
